src/um/avrt.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2016 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+
+use shared::minwindef::{BOOL, LPDWORD, PULONG};
+
+use shared::guiddef::GUID;
+
+use um::winnt::{HANDLE, PLARGE_INTEGER, LONGLONG, PHANDLE, LPCWSTR, LPCSTR};
+
+ENUM!{enum AVRT_PRIORITY {
+    AVRT_PRIORITY_VERYLOW = -2i32 as u32,
+    AVRT_PRIORITY_LOW = -1i32 as u32,
+    AVRT_PRIORITY_NORMAL = 0u32,
+    AVRT_PRIORITY_HIGH = 1u32,
+    AVRT_PRIORITY_CRITICAL = 2u32,
+}}
+
+pub const THREAD_ORDER_GROUP_INFINITE_TIMEOUT: LONGLONG = -1;
+
+FN!(stdcall AvSetMmThreadCharacteristicsA(TaskName: LPCSTR,
```

Comment:
`FN!` is the wrong macro to use. You should be using `EXTERN!` here. Also the indentation is wrong.
https://github.com/retep998/winapi-rs/blob/dev/CONTRIBUTING.md#functions